Products/CMFPlone/controlpanel/browser/recyclebin.py

```diff
@@ -0,0 +1,46 @@
+# testcp.py
+from plone.app.registry.browser.controlpanel import ControlPanelFormWrapper
+from plone.app.registry.browser.controlpanel import RegistryEditForm
+from plone.z3cform import layout
+from zope import schema
+from zope.interface import Interface
+
+
+class IRecycleBinControlPanelSettings(Interface):
+    recycling_enabled = schema.Bool(
+        title="Enable the Recycle Bin",
+        description="Enable or disable the Recycle Bin feature.",
+        default=True,
+    )
+
+    retention_period = schema.Int(
+        title="Retention Period",
+        description="Number of days to keep deleted items in the Recycle Bin.",
+        default=30,
+        min=1,
+    )
+
+    maximum_size = schema.Int(
+        title="Maximum Size",
+        description="Maximum size of the Recycle Bin in MB.",
+        default=100,
+        min=10,
+    )
+
+    auto_purge = schema.Bool(
+        title="Auto Purge",
+        description="Automatically purge items older than the retention period.",
+        default=True,
+    )
```

Comment:
If this is False, then the retention period setting can be misleading, since it doesn't actually do anything then.

Maybe just use the retention_period setting, and have users set it to 0 if they don't want to auto-purge?

Reply:
done in https://github.com/plone/Products.CMFPlone/pull/4168/commits/df30fefe20436a5edac7cbdeb2fce8270a1a0637